courses/models.py

```diff
@@ -664,6 +664,19 @@ class Meta:
     def get_certified_object_id(self):
         return self.course_run_id
 
+    @property
+    def link(self):
```

Comment:
what about `certificate_link` ?

Reply:
It's already in the `Certificate` object so I felt that would be redundant. As in, `Certificate.certificate_link` and `Certificate.link`, I opted for the latter.